packages/frontend-shared/src/gql-components/HeaderBarContent.vue

```diff
@@ -265,7 +265,7 @@ fragment HeaderBar_HeaderBarContent on Query {
     branch
     isLoadingNodeEvents
   }
-  projectRootFromCI
+  isInGlobalMode,
```

Comment:
Hi @estrada9166 ! It looks like you added `projectRootFromCI` in [this PR from March](https://github.com/cypress-io/cypress/pull/20470). Was there a reason you created `projectRootFromCI` instead of using `isInGlobalMode`?